### PR TITLE
task DEV-2

### PR DESCRIPTION
**Pull request by OlgaYurkova**

### CODE REVIEW

File: DEV-2/SearchForEvenIndices.cs [hunk 1]
```diff
@@ -0,0 +1,32 @@
+﻿using System;
+using System.Text;
+
```
Comment (IgorKheidorov) [inline DEV-2/SearchForEvenIndices.cs:3]: Empty line

File: DEV-2/SearchForEvenIndices.cs [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+{
+    /// <summary>
+    ///The SequenceSearch class is the main class of the program that works with the string
+    ///Outputs to the console the original string consisting of characters with even indices
+    /// </summary>
+    class SearchForEvenIndices
+    {
+        static void Main(string[] args)
+        {
+
```
Comment (IgorKheidorov) [inline DEV-2/SearchForEvenIndices.cs]: code style

File: DEV-2/SearchForEvenIndices.cs [hunk 3]
```diff
@@ -0,0 +1,32 @@
...
+            {
+                Console.WriteLine("The string wasn't entered");
+            }
+
+            for (int i = 1; i < inputString.Length; i++)
+            {
+                inputString.Remove(i, 1);
+            }
+            Console.WriteLine(inputString);
+
```
Comment (IgorKheidorov) [inline DEV-2/SearchForEvenIndices.cs]: code style

File: DEV-2/SearchForEvenIndices.cs [hunk 4]
```diff
@@ -0,0 +1,32 @@
+﻿using System;
+using System.Text;
+
+
+namespace DEV_2
+{
+    /// <summary>
+    ///The SequenceSearch class is the main class of the program that works with the string
```
Comment (IgorKheidorov) [inline DEV-2/SearchForEvenIndices.cs]: Comment do not match class name

**Review by IgorKheidorov — CHANGES_REQUESTED** [review]

File: DEV-2/SearchForEvenIndices.cs [hunk 5]
```diff
@@ -0,0 +1,29 @@
+﻿using System;
+using System.Text;
+namespace DEV_2
+{
+    /// <summary>
+    ///The SearchForEvenIndices class is the main class of the program that works with the string
```
Comment (AlexanderSoroka) [inline DEV-2/SearchForEvenIndices.cs]: missed spaces after ///

File: DEV-2/SearchForEvenIndices.cs [hunk 6]
```diff
@@ -0,0 +1,29 @@
+﻿using System;
+using System.Text;
+namespace DEV_2
```
Comment (AlexanderSoroka) [inline DEV-2/SearchForEvenIndices.cs]: missed blank line

File: DEV-2/SearchForEvenIndices.cs [hunk 7]
```diff
@@ -0,0 +1,29 @@
...
+            StringBuilder inputString = new StringBuilder(Console.ReadLine());
+
+            for (int i = 1; i < inputString.Length; i++)
+            {
+                inputString.Remove(i, 1);
+            }
+
+            if (inputString.Length == 0)
+            {
+                Console.WriteLine("The string wasn't entered");
```
Comment (AlexanderSoroka) [inline DEV-2/SearchForEvenIndices.cs]: incorrect end user message

**Review by AlexanderSoroka — CHANGES_REQUESTED** [review]

**Pull request merged**